tests/phpunit/CRM/Upgrade/Incremental/php/SixOneTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Civi\Api4\UserJob;
+
+/**
+ * Class CRM_Upgrade_Incremental_php_SixOneTest
+ * @group headless
+ */
+class CRM_Upgrade_Incremental_php_SixOneTest extends CiviUnitTestCase {
+
+  public function testUpdateUserJobs(): void {
+    $userJobParameters = [
+      'metadata' => [
+        'DataSource' => ['table_name' => 'abc', 'column_headers' => ['External Identifier', 'Amount Given', 'Contribution Date', 'Financial Type', 'In honor']],
+        'submitted_values' => [
+          'contactType' => 'Individual',
+          'contactSubType' => '',
+          'dataSource' => 'CRM_Import_DataSource_SQL',
+          'onDuplicate' => CRM_Import_Parser::DUPLICATE_SKIP,
+          'dedupe_rule_id' => NULL,
+          'dateFormats' => CRM_Utils_Date::DATE_yyyy_mm_dd,
+        ],
+        'import_mappings' => [
+          ['name' => 'external_identifier'],
+          ['name' => 'total_amount'],
+          ['name' => 'receive_date'],
+          ['name' => 'financial_type_id'],
+          ['name' => 'contribution_source'],
+        ],
+      ],
+      'status_id:name' => 'draft',
+      'job_type' => 'contribution_import',
+    ];
+    $userJobID = UserJob::create()->setValues($userJobParameters)->execute()->first()['id'];
+    CRM_Upgrade_Incremental_php_SixOne::upgradeImportMappingFields(NULL, 'upgrade_6_1_beta1', 'Import Contribution');
```

Comment:
your passing 3 arguments into this function when it is only expecting 2 https://github.com/civicrm/civicrm-core/blob/master/CRM/Upgrade/Incremental/php/SixOne.php#L49

Reply:
I'm generous like that